src/timer-view/timer-view.html

```diff
@@ -14,7 +14,7 @@
     [[prompt]]
     <div>
       [[secondsRemaining]]
-      <iron-timer id="timer" start-time="1" current-time="{{secondsRemaining}}" on-iron-timer-end="_timerEnd"></iron-timer>
```

Comment:
That knowledge isn't represented anywhere. That's the kind of thing that should at the very least go into the commit comment, so that someone can use the "blame" feature to see why a specific line of code is there. Worst case, it should be a comment, if there's no other way to design around it.